core/loader/utils.ts

```diff
@@ -1,11 +1,19 @@
 import type { TextResponse } from '../download.js'
 import type { PreviewCandidate } from '../preview.js'
 
+export const HREF_PATTERNS = {
```

Comment:
Why we should have patterns here? Let’s just put string to `loader/rss.ts` and `loader/atom.ts`

Reply:
I suggested this because I use the string in two places, [here](https://github.com/hplush/slowreader/pull/159/files#diff-f3c4dfecd2578ed8df8411f53596fa80abd2ffea573bf8717bd2a5a7f9abcacdR70) and [here](https://github.com/hplush/slowreader/pull/159/files#diff-b25629b68bfa446d7ccfa913ef99c6abfa67d78b8f8823f1715b353937f9c969R30) , but ok, I'll redo it